v3/lints/cabf_cs_br/lint_cs_crl_distribution_points.go

```diff
@@ -0,0 +1,56 @@
+package cabf_cs_br
+
+import (
+	"strings"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+/*7.1.2.3 b. cRLDistributionPoints
+This extension MUST be present. It MUST NOT be marked critical, and it MUST contain the
+HTTP URL of the CA’s CRL service*/
+
+func init() {
+	lint.RegisterCertificateLint(&lint.CertificateLint{
+		LintMetadata: lint.LintMetadata{
+			Name:          "e_cs_crl_distribution_points",
+			Description:   "This extension MUST be present. It MUST NOT be marked critical.",
+			Citation:      "CABF CS BRs 7.1.2.3.b",
+			Source:        lint.CABFCSBaselineRequirements,
+			EffectiveDate: util.CABF_CS_BRs_1_2_Date,
+		},
+		Lint: NewCrlDistributionPoints,
+	})
+}
+
+type crlDistributionPoints struct{}
+
+func NewCrlDistributionPoints() lint.LintInterface {
+	return &crlDistributionPoints{}
+}
+
+func (l *crlDistributionPoints) CheckApplies(c *x509.Certificate) bool {
+	return util.IsCodeSigning(c.PolicyIdentifiers) && util.IsSubscriberCert(c)
```

Comment:
Based on the preamble to the document...

> The scope of these Requirements includes all "Code Signing Certificates", as defined below, and associated Timestamp Authorities, and all Certification Authorities technically capable of issuing Code Signing Certificates, including any Root CA that is publicly trusted for code signing and all other CAs that might serve to complete the validation path to such Root CA.

...I believe that we can safely add the following assumption to [base.go](https://github.com/zmap/zlint/blob/672100d27702f1ddef95ecaa8977b3b1ea0f4b2e/v3/lint/base.go#L220).

```go
	if l.Source == CABFCSBaselineRequirements && !util.IsCodeSigning(cert) {
		return &LintResult{Status: NA}
	}
```

Additionally, I am somewhat indecisive about the `IsSubscriber` clause that we have attached to these specific lints. It's unfortunate, but the document seems to have a mix of explicitly stating when a set of requirements is target specifically subscriber certs as well as implicit. That is, lints such as these seem like they would naturally apply to subscriber certs, but it's not explicitly spelled out like I am seeing in some other sections...unless I am simply not seeing it.

Reply:
Looking at the CABF CS BR 7.1.2, these lints are targeted at the Subscriber Certificate profile, there is some overlap with the Sub CA profile. but none of these lints really work with the Root CA profile.
```
e_cs_crl_distribution_points
Applies to: Both Subscriber and Subordinate CA certificates. (7.1.2.2b and 7.1.2.3b)
Root CA: Not explicitly required.

e_cs_eku_required
Applies to: Both Subscriber and Subordinate CA certificates.
Root CA: EKUs MUST NOT be present. (7.1.2.1d)
Subordinate CA: Also prohibits id-kp-emailProtection. (7.1.2.2f)

e_cs_key_usage_required
Applies to: Specifically to Subscriber certificates.
Subordinate CA and Root CA: Both require cRLSign and keyCertSign, whereas Code Signing Certificates prohibits both. 

e_cs_rsa_key_size
Subscriber Certificates: RSA key size MUST be at least 3072 bits.
Root and Subordinate CA Certificates: RSA key size MUST be at least 4096 bits.
```

So we could remove the IsSubscriber check, I would need to update the lints to accommodate the other profiles